crate_universe/private/crates_vendor.bzl

```diff
@@ -7,27 +7,43 @@ load("//rust/platform:triple_mappings.bzl", "SUPPORTED_PLATFORM_TRIPLES")
 
 _UNIX_WRAPPER = """\
 #!/usr/bin/env bash
+
 set -euo pipefail
+
 export RUNTIME_PWD="$(pwd)"
 if [[ -z "${{BAZEL_REAL:-}}" ]]; then
     BAZEL_REAL="$(which bazel || echo 'bazel')"
 fi
 
+_ENVIRON=()
+_ENVIRON+=(BAZEL_REAL="${{BAZEL_REAL}}")
+_ENVIRON+=(BUILD_WORKSPACE_DIRECTORY="${{BUILD_WORKSPACE_DIRECTORY}}")
+_ENVIRON+=(PATH="${{PATH}}")
+
+if [[ -n "${{CARGO_BAZEL_DEBUG:-}}" ]]; then
+    _ENVIRON+=(CARGO_BAZEL_DEBUG="${{CARGO_BAZEL_DEBUG}}")
+fi
+
 # The path needs to be preserved to prevent bazel from starting with different
 # startup options (requiring a restart of bazel).
 # If you provide an empty path, bazel starts itself with
 # --default_system_javabase set to the empty string, but if you provide a path,
 # it may set it to a value (eg. "/usr/local/buildtools/java/jdk11").
-exec env - BAZEL_REAL="${{BAZEL_REAL}}" BUILD_WORKSPACE_DIRECTORY="${{BUILD_WORKSPACE_DIRECTORY}}" PATH="${{PATH}}" {env} \\
-"{bin}" {args} "$@"
+exec env - \\
+${{_ENVIRON[@]}} \\
```

Comment:
I think this array expansion need to be quoted in order to properly handle paths that contain spaces.

Reply:
https://github.com/bazelbuild/rules_rust/pull/3078